sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -28,9 +28,8 @@
         <span>
           <font-awesome-icon :icon="['fas', 'map-pin']" />
           <span class="category-header-medium mx-2">{{ item.name }}</span>
-          <designation-and-restriction-status-badge
-            class="mb-4 ml-4"
-            status="designated"
+          <status-chip-designation-agreement-location
```

Comment:
Can we have it renamed to only `status-chip-designation-agreement` since it is used for both location and institution statuses?